packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -28,104 +40,71 @@ const _testTags: ITag[] = [
-          }}
-          itemLimit={2}
-          disabled={this.state.isPickerDisabled}
-          inputProps={{
-            onBlur: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called'),
-            onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called'),
-            'aria-label': 'Tag Picker',
-          }}
-        />
-        <br />
-        Filter items on selected: This picker will show already-added suggestions but will not add duplicate tags.
-        <TagPicker
-          removeButtonAriaLabel="Remove"
-          componentRef={this._picker}
-          onResolveSuggestions={this._onFilterChangedNoFilter}
-          onItemSelected={this._onItemSelected}
-          getTextFromItem={this._getTextFromItem}
-          pickerSuggestionsProps={{
-            suggestionsHeaderText: 'Suggested Tags',
-            noResultsFoundText: 'No Color Tags Found',
-          }}
-          itemLimit={2}
-          disabled={this.state.isPickerDisabled}
-          inputProps={{
-            onBlur: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called'),
-            onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called'),
-            'aria-label': 'Tag Picker',
-          }}
-        />
-      </div>
-    );
-  }
-
-  private _getTextFromItem(item: ITag): string {
-    return item.name;
-  }
-
-  private _onDisabledButtonClick = (): void => {
-    this.setState({
-      isPickerDisabled: !this.state.isPickerDisabled,
-    });
-  };
+  const listContainsDocument = useConstCallback((tag: ITag, tagList?: ITag[]) => {
+    if (!tagList || !tagList.length || tagList.length === 0) {
+      return false;
+    }
+    return tagList.filter(compareTag => compareTag.key === tag.key).length > 0;
+  });
 
-  private _onFilterChanged = (filterText: string, tagList: ITag[]): ITag[] => {
+  const onFilterChanged = useConstCallback((filterText: string, tagList: ITag[]): ITag[] => {
     return filterText
-      ? _testTags
+      ? testTags
           .filter(tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0)
-          .filter(tag => !this._listContainsDocument(tag, tagList))
+          .filter(tag => !listContainsDocument(tag, tagList))
       : [];
-  };
+  });
 
-  private _onFilterChangedNoFilter = (filterText: string, tagList: ITag[]): ITag[] => {
-    return filterText ? _testTags.filter(tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0) : [];
-  };
+  const onFilterChangedNoFilter = useConstCallback((filterText: string, tagList: ITag[]): ITag[] => {
+    return filterText ? testTags.filter(tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0) : [];
+  });
 
-  private _onItemSelected = (item: ITag): ITag | null => {
-    if (this._picker.current && this._listContainsDocument(item, this._picker.current.items)) {
-      return null;
-    }
-    return item;
-  };
+  const onItemSelected = React.useCallback(
```

Comment:
You can do `useConstCallback` here since the only dependency is another const callback.